endpoints/profiles.md

```diff
@@ -35,3 +61,118 @@ The profile contains the contact information as well as the branding information
 * ```GET /profiles.json``` returns all profiles for the user
```

Comment:
I believe the `.json` format is optional here. @ryansch correct me if I'm wrong.

Reply:
@morganick you're correct